README.md

```diff
@@ -128,18 +110,13 @@ bytestream: 25 03 00 00 00
 
 ```
 
-## HOW TO BUILD Z/IP/GATEWAY
```

Comment:
shouldn´t we keep this chapter title

Reply:
It is the same title as the previous one. Did you have another idea?

Reply:
good point, I thought it was exactly the same title as the previous one.